src/components/SupportingPartners/index.js

```diff
@@ -0,0 +1,82 @@
+import { A } from "@commons-ui/core";
+import { Typography } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import Image from "next/image";
+import PropTypes from "prop-types";
+import React from "react";
+
+import Section from "@/pesayetu/components/Section";
+
+const useStyles = makeStyles(({ palette, typography }) => ({
+  root: {
+    paddingTop: typography.pxToRem(56.69),
+    paddingBottom: typography.pxToRem(80),
+    background: palette.grey.light,
+  },
+  title: {
+    fontWeight: 900,
+    color: palette.grey.dark,
+    marginBottom: typography.pxToRem(49.38),
+  },
+  logoContainer: {
+    display: "flex",
+    alignItems: "center",
+    justifyContent: "center",
+    flexWrap: "wrap",
+  },
```

Comment:
Aren't you just recreating `Grid`?

Reply:
yes because the items are 7 and I couldn't find a way to divide 12 into 7 wholes. 😅 @kilemensi